store/data-provider/index.js

```diff
@@ -32,6 +32,8 @@ class DataProvider extends Resource {
     recommender: null,
   }
 
+  @observable location = {}
```

Comment:
Why is this needed? Does not `extendObservable` in the Resource.extend() handle this?

Reply:
Sadly not. I guess it has something to do with the first render. (variable is not observable at that moment)

It's needed for hiding regional warning on /deposit-compliance, i.e. if you access repository as a super admin the warning is shown becuase we don't have `location.countryCode` yet. This observable makes it rerender (hide) once it's loaded.